Modules/selectmodule.c

```diff
@@ -1031,35 +1058,21 @@ devpoll_get_closed(devpollObject *self)
         Py_RETURN_FALSE;
 }
 
-static PyObject*
-devpoll_fileno(devpollObject *self)
+/*[clinic input]
+select.devpoll.fileno
```

Comment:
Please try to use "-> int" return converter. Same comment for other fileno() methods.

Reply:
Regarding `devpoll.fileno()`, due to the use of `devpoll_err_closed()` internally, using `-> int` or `-> long` actually complicates the code. The same goes for `epoll.fileno()` and `kqueue.fileno()` as well.

Reply:
This will complicate the code. `select_devpoll_fileno_impl` would be need to convert NULL returned by `devpoll_err_closed()` to -1. I think it is better to keep the current code and add the converter in a separate issue if you with.